examples/react-apollo-crm/pnpm-lock.yaml

```diff
@@ -0,0 +1,4938 @@
+lockfileVersion: 5.4
```

Comment:
why was the lockfile generated? it is supposed to use the lock file in the monorepo root. The other examples don't have any lockfile to avoid having to maintain it every time we release new versions or change the underlying dependencies. I would remove this